java/spark/pom.xml

```diff
@@ -51,6 +51,45 @@
                     </args>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <version>3.6.1</version>
+                <executions>
+                    <execution>
+                        <id>copy-dependencies</id>
```

Comment:
It would be better to split these changes into a new PR?

Reply:
OK